locales/make_locale_x64.bat

```diff
@@ -70,72 +71,96 @@ if "%BUILD_MODE%" == "." set BUILD_MODE=release
 
 @echo. Searching for Visual Studio installs...
 
-if NOT "%VS80COMNTOOLS%"=="" (
-@echo. Found installation for Visual Studio 2005
+@rem 1. search Visual Studio installation (from newest to oldest)
+@echo checking Visual Studio 2017 v150... VS150COMNTOOLS = "%VS150COMNTOOLS%"
+if defined VS150COMNTOOLS (
+    @echo. Found installation for Visual Studio 2017 150: "%VS150COMNTOOLS%"
+    if exist "%VS150COMNTOOLS%VsDevCmd.bat" (
+        echo Found %BUILD_TARGET% configuration in Visual Studio 2017_150 Community.
+        @echo call "%VS150COMNTOOLS%VsDevCmd.bat" -arch=amd64
+        call "%VS150COMNTOOLS%VsDevCmd.bat" -arch=amd64
+        goto :BUILD
+    )
+    @rem it should be done similar for Professional and Enterprise editions (like for VS2017v140) but I have access to none of them to be sure about the paths
+    @rem configuration for this VS version didn't work, try older VS versions...
 )
-if NOT "%VS90COMNTOOLS%"=="" (
-@echo. Found installation for Visual Studio 2008
+@echo checking Visual Studio 2017 v150... VS150COMCOMNTOOLS = "%VS150COMCOMNTOOLS%"
+if defined VS150COMCOMNTOOLS (
+    @echo. Found installation for Visual Studio 2017 150: "%VS150COMCOMNTOOLS%"
+    if exist "%VS150COMCOMNTOOLS%VsDevCmd.bat" (
+        echo Found %BUILD_TARGET% configuration in Visual Studio 2017_150 Community.
+        @echo call "%VS150COMCOMNTOOLS%VsDevCmd.bat" -arch=amd64
+        call "%VS150COMCOMNTOOLS%VsDevCmd.bat" -arch=amd64
+        goto :BUILD
+    )
+    @rem it should be done similar for Professional and Enterprise editions (like for VS2017v140) but I have access to none of them to be sure about the paths
+    @rem configuration for this VS version didn't work, try older VS versions...
 )
-if NOT "%VS100COMNTOOLS%"=="" (
-@echo. Found installation for Visual Studio 2010
+@echo checking Visual Studio 2017 v140... VS140COMNTOOLS = "%VS140COMNTOOLS%"
+if defined VS140COMNTOOLS (
+    @echo. Found installation for Visual Studio 2017 v140: "%VS140COMNTOOLS%"
+    @echo checking "%VS140COMNTOOLS%..\..\..\..\2017\Community\Common7\ToolsVsDevCmd.bat"

```

Comment:
please confirm `ToolsVsDevCmd.bat` and notice there are a few same references.
I think it is `Tools\VsDevCmd.bat`

Reply:
fixed